backend/api/migrations/0011_credit_trade_category.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.23 on 2023-11-06 23:13
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0010_migrate_compliance_report'),
+    ]
+
+    operations = [
+        migrations.RunSQL("UPDATE credit_trade SET trade_category_id = NULL WHERE type_id != 1 AND type_id != 2")
```

Comment:
Are we certain these ids are consistent across environments?

Reply:
updated the script to search for "Buy" and "Sell" values in the Credit_trade_type table for better reliability of updating the correct values.

`UPDATE credit_trade SET trade_category_id = null FROM credit_trade_type WHERE credit_trade.trade_category_id IS NOT NULL AND credit_trade_type.id = credit_trade.trade_category_id AND credit_trade_type.the_type NOT IN ('Buy', 'Sell');`